Resources/public/js/views/fields/ez-country-editview.js

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-country-editview', function (Y) {
+    "use strict";
+    /**
+     * Provides the field edit view for the country fields
+     *
+     * @module ez-country-editview
+     */
+
+    Y.namespace('eZ');
+
+    var FIELDTYPE_IDENTIFIER = 'ezcountry',
+        IS_LIST_HIDDEN = 'is-list-hidden';
+
+    /**
+     * Country edit view
+     *
+     * @namespace eZ
+     * @class CountryEditView
+     * @constructor
+     * @extends eZ.SelectionEditView
+     */
+    Y.eZ.CountryEditView = Y.Base.create('countryEditView', Y.eZ.SelectionEditView, [], {
+        VALUES_TPL: '<li class="ez-selection-value" />',
+
+        /**
+         * Default implementation of the field edit view render. By default, it
+         * injects the field definition, the field, the content and the content
+         * type
+         *
+         * @method render
+         * @return {eZ.FieldEditView}
+         */
+        render: function () {
+            this.get('container').addClass(IS_LIST_HIDDEN);
+            var defaultVariables = {
+                fieldDefinition: this.get('fieldDefinition'),
+                field: this.get('field'),
+                content: this.get('content').toJSON(),
+                version: this.get('version').toJSON(),
+                contentType: this.get('contentType').toJSON()
+            };
+
+            this.get('container').setHTML(
+                this.template(Y.mix(this._variables(), defaultVariables, true))
+            );
+            return this;
+        },
+
+        /**
+         * Returns the current country name.
+         * If the country is missing, it will return a country code and send a warning.
+         *
+         * @method _getCountryName
+         * @param {String} alpha2Code The country code (Alpha2)
+         * @protected
+         * @return {String}
+         */
+        _getCountryName: function (alpha2Code) {
+            var config = this.get('config');
+
+            if (config[alpha2Code]) {
+                return config[alpha2Code].Name;
+            } else {
+                console.warn("Unknown country code: " + alpha2Code);
+                return alpha2Code;
+            }
+        },
+
+        _initSelectionFilter: function () {
```

Comment:
to avoid overriding this whole method, you should probably change the arguments of `_addSelection`/`_removeSelection`